src/common/PeopleTile/styled.js

```diff
@@ -3,7 +3,7 @@ import styled, { css } from "styled-components";
 export const StyledTile = styled.div`
   display: flex;
   flex-direction: column;
-  width: 248px;
+  width: 208px;
```

Comment:
Tutaj ten width był większy, niż mogłoby to wynikać z Figmy, ponieważ z tego co pamiętam umawialiśmy, że skoro API na jedną stronę popularPeople daje 20 wyników to ustawimy to sobie w czterech rzędach i pięciu kolumnach, dlatego zdecydowałem się proporcjonalnie powiększyć te kafelki ;)
Stąd właśnie ta wartość nie jest uniwersalna, tak więc nie wiem czy zostawiamy 208 i rezygnujemy z tego ustawiania, o którym mówiliśmy w popular People czy jednak dzielimy to na dwa rozmiary np. za pomocą jakiegoś propsa?

Reply:
Ok, jakoś nie zarejestrowałam tej zmiany, wybacz 🙈 to cofnę to i zostawimy wszędzie te kafelki większe żeby było spójnie :)